developer_setup.md

```diff
@@ -2,39 +2,14 @@
 
 ### Install System Packages
 
-#### Fedora 20+
+#### Fedora / CentOS 7
 
-* Install packages
-
-  ```bash
-  sudo yum -y install git-all                            # Git and components
-  sudo yum -y install memcached                          # Memcached for the session store
-  sudo yum -y install postgresql-devel postgresql-server # PostgreSQL Database server and to build 'pg' Gem
-  sudo yum -y install libxml2-devel libxslt-devel        # For Nokogiri Gem
-  sudo yum -y install gcc-c++                            # For event-machine Gem
-  ```
-
-* Enable Memcached
-
-  ```bash
-  sudo systemctl enable memcached
-  sudo systemctl start memcached
-  ```
-
-* Configure PostgreSQL
+* Enable EPEL (CentOS only)
 
   ```bash
-  sudo passwd postgres <new_password>
-  su postgres -c 'initdb -D /var/lib/pgsql/data'
-  sudo systemctl enable postgresql
-  sudo systemctl start postgresql
-  su postgres -c "psql -c \"CREATE ROLE root SUPERUSER LOGIN PASSWORD 'smartvm'\""
+  sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
No dnf in centos as of 7.2.1511 (Core) but it can be installed from EPEL, maybe say `Enable EPEL and install DNF (CentOS only)`?